src/main/java/javaserver/RequestHandlers/FileHandler.java

```diff
@@ -0,0 +1,18 @@
+package javaserver.RequestHandlers;
+
+import javaserver.Request;
+import javaserver.Routes;
+
+public class FileHandler extends RequestHandler {
+
```

Comment:
Another issue with consistency here. Seems like in some files, you add a newline between the class declaration and the first method/property, and in others you don't.